src/app.js

```diff
@@ -10,83 +11,99 @@ import errorLogger from './middleware/errorLogger.js';
-                    next({
-                        message: 'A verifiable credential must be provided in the body',
-                        code: 400
-                    })
-               }
-                const vcWithStatus = await allocateStatus(vc)
-                return res.json(vcWithStatus)
-            } catch (e) {
-                 // We catch the async errors and pass them to the error handler.
-                if (!e.message) {e.message = "Error when allocating status position."}
-                // Note that if e contains a code property, the following spread of e will
-                // (correctly) overwrite the 500
-                next({code: 500, ...e})
-            }
-        })
-
-    // the body will look like:  {credentialId: '23kdr', credentialStatus: [{type: 'StatusList2021Credential', status: 'revoked'}]}
-    app.post("/credentials/status",
-        async (req, res, next) => {
-            try {
-                const updateRequest = req.body
-                if (!updateRequest || !updateRequest.credentialId || !updateRequest.credentialStatus) {
-                    next({
-                        message: 'A status update request must be provided in the body',
-                        code: 400
-                    })
-               }
-                const { credentialId, credentialStatus } = updateRequest
-                const status = credentialStatus[0].status
-                const statusType = credentialStatus[0].type
+  app.get('/', function (req, res, next) {
+    res.send({ message: 'status-service server status: ok.' });
+  });
 
-                if (statusType !== 'StatusList2021Credential') {
-                    next({
-                        message: 'StatusList2021Credential is the only supported status type.',
-                        code: 400
-                    })
-                }
-                const statusResponse = await revoke(credentialId, status)
-                return res.status(statusResponse.code).send(statusResponse)
-
-            } catch (e) {
-                 // We catch the async errors and pass them to the error handler.
-                if (!e.message) {e.message = "Error updating credential status position."}
-                // Note that if e contains a code property, the following spread of e will
-                // (correctly) overwrite the 500
-                next({code: 500, ...e})
-            }
-        })
+  // get status credential
+  app.get('/:statusCredentialId', async (req, res) => {
+    if (credStatusService !== 'mongodb') {
+      return null;
+    }
+    const statusCredentialId = req.params.statusCredentialId;
+    let errorMessage;
+    try {
+      const statusCredential = await status.getStatusCredential(statusCredentialId);
+      if (!statusCredential) {
+        errorMessage = `Unable to find Status Credential with ID "${statusCredentialId}".`;
+        return res.status(404).send(errorMessage);
```

Comment:
We should probably talk about the error handling. I had tried to use the express handling which basically passes the error along to the handler (using next({code, message)) rather than returning the error directly on the res. One reason for that is to consistently  log all errors - the handler first logs the error before returning it. But, happy to talk about this if there is a better way.

Reply:
OK, let's talk about this today or tomorrow.

Reply:
Done ✅